Updated Karthik/Stacey roles, blog post draft 🫡 (WIP) (#746)
ropensci/roweb3: content/blog/2024-03-29-hello-from-our-new-executive-director/index.es.md

---
title: Hola de nuestro nuevo Director Ejecutivo
slug: hello-from-our-new-executive-director
author:
- Karthik Ram
date: '2024-03-29'
tags:
- transiciones
- liderazgo
featured: true
description: Noam Ross toma el timón como nuevo Director Ejecutivo de rOpenSci
---

> Me complace, emociona y llena de humildad anunciar que asumo el cargo de Director Ejecutivo de rOpenSci a partir del 1 de abril.

Reviewer: ```suggestion
Me complace, emociona y llena de humildad anunciar que asumo el cargo de Director Ejecutivo de rOpenSci a partir del 1 de Abril.
```

En primer lugar, permítanme expresar mi gratitud a nuestro Director Ejecutivo saliente y amigo Karthik Ram por su liderazgo y tutoría al frente de rOpenSci durante la última década.  Ha sido una mano firme y un visionario que ha ayudado a nuestra comunidad a lograr tanto juntos en este tiempo.  Gracias a él y al resto de [equipo](https://ropensci.org/about/#team) por confiarme este papel.

He tenido el placer de ser miembro de rOpenSci durante muchos años: como voluntario, editor de la revisión por pares de software de rOpenSci desde 2015 y director del programa de revisión por pares desde 2019.  Soy [ecólogo de enfermedades en una ONG](https://www.ecohealthalliance.org/personnel/dr-noam-ross) donde trabajo todos los días con las herramientas de análisis reproducible que crea la comunidad rOpenSci.

> Estoy profundamente comprometida con los valores de rOpenSci de investigación abierta y reproducible utilizando datos compartidos, de creación de infraestructuras y de una comunidad acogedora e integradora que permita a investigadores y desarrolladores apoyarse mutuamente para realizar investigaciones impactantes y trabajar por el bien público.

Reviewer: ```suggestion
Estoy profundamente comprometida con los valores de rOpenSci de investigación abierta y reproducible utilizando datos compartidos, de creación de infraestructuras y de una comunidad acogedora e integradora que permita a quienes investigan y desarrollan software apoyarse mutuamente para realizar investigaciones impactantes y trabajar por el bien público.
```

> Como director, estoy deseando trabajar con nuestra enorme comunidad de investigadores, desarrolladores, revisores, editores, mentores, campeones y personal.  Mi objetivo es fortalecer los pilares de nuestro trabajo y ampliarlos a través de nuevas asociaciones: incorporar nuevas voces y talentos al ecosistema de código abierto, mejorar el software de investigación mediante la revisión por pares, crear redes de apoyo mutuo entre desarrolladores de software de investigación y crear nuevas infraestructuras y herramientas en la frontera de la práctica reproducible.

Reviewer: ```suggestion
Como director, estoy deseando trabajar con nuestra enorme comunidad de personas que investigan, desarrollan, revisan, editan, son parte de nuestro programa de campeones campeonas y del staff.  Mi objetivo es fortalecer los pilares de nuestro trabajo y ampliarlos a través de nuevas asociaciones: incorporar nuevas voces y talentos al ecosistema de código abierto, mejorar el software de investigación mediante la revisión por pares, crear redes de apoyo mutuo entre quienes desarrollan software de investigación y crear nuevas infraestructuras y herramientas en la frontera de la práctica reproducible.
```

> Sé que esta comunidad seguirá haciendo grandes cosas juntos, ¡y me alegro de poder formar parte de ella!

Reviewer: ```suggestion
Sé que esta comunidad seguirá haciendo grandes cosas, ¡y me alegro de poder formar parte de ella!
```

> [Lea el mensaje de despedida del director fundador Karthik Ram](https://ropensci.org/blog/2024/03/29/from-the-founding-director-my-farewell-to-ropensci/)

Reviewer: ```suggestion
[Lee el mensaje de despedida del director fundador Karthik Ram](https://ropensci.org/blog/2024/03/29/from-the-founding-director-my-farewell-to-ropensci-es/)
```

---
title: Hola de nuestro nuevo Director Ejecutivo
slug: hello-from-our-new-executive-director
author:
- Noam Ross
date: '2024-03-29'
tags:
- transiciones
- liderazgo
featured: true
description: Noam Ross toma el timón como nuevo Director Ejecutivo de rOpenSci
preface: "[Read this post in English](blog/2024/03/29/hello-from-our-new-executive-director/)"
---

Me complace, emociona y llena de humildad anunciar que asumo el cargo de Director Ejecutivo de rOpenSci a partir del 1 de abril.

> En primer lugar, permítanme expresar mi gratitud a nuestro Director Ejecutivo saliente y amigo Karthik Ram por su liderazgo y tutoría al frente de rOpenSci durante la última década.  Ha sido una mano firme y un visionario que ha ayudado a nuestra comunidad a lograr tanto juntos en este tiempo.  Gracias a él y al resto de [equipo](https://ropensci.org/about/#team) por confiarme este papel.

Reviewer 2: ```suggestion
En primer lugar, permítanme expresar mi gratitud a nuestro Director Ejecutivo saliente y amigo Karthik Ram por su liderazgo y tutoría al frente de rOpenSci durante la última década.  Ha sido una mano firme y un visionario que ha ayudado a nuestra comunidad a lograr tanto juntos en este tiempo.  Gracias a él y al resto de [equipo](/about/#team) por confiarme este papel.
```

Estoy profundamente comprometida con los valores de rOpenSci de investigación abierta y reproducible utilizando datos compartidos, de creación de infraestructuras y de una comunidad acogedora e integradora que permita a investigadores y desarrolladores apoyarse mutuamente para realizar investigaciones impactantes y trabajar por el bien público.

Como director, estoy deseando trabajar con nuestra enorme comunidad de investigadores, desarrolladores, revisores, editores, mentores, campeones y personal.  Mi objetivo es fortalecer los pilares de nuestro trabajo y ampliarlos a través de nuevas asociaciones: incorporar nuevas voces y talentos al ecosistema de código abierto, mejorar el software de investigación mediante la revisión por pares, crear redes de apoyo mutuo entre desarrolladores de software de investigación y crear nuevas infraestructuras y herramientas en la frontera de la práctica reproducible.

Sé que esta comunidad seguirá haciendo grandes cosas juntos, ¡y me alegro de poder formar parte de ella!

> [Lea el mensaje de despedida del director fundador Karthik Ram](https://ropensci.org/es/blog/2024/03/29/from-the-founding-director-my-farewell-to-ropensci-es/)

Reviewer 2: ```suggestion
[Lea el mensaje de despedida del director fundador Karthik Ram](/blog/2024/03/29/from-the-founding-director-my-farewell-to-ropensci-es/)
```
